modular_citadel/code/modules/reagents/chemistry/reagents/healing.dm

```diff
@@ -136,6 +137,10 @@
 		return ..()
 	if(passed_data["grown_volume"] > data["grown_volume"])
 		data["grown_volume"] = passed_data["grown_volume"]
+	message_admins("merge: data: [data["injected_vol"]] passed: [passed_data["injected_vol"]]")
```

Comment:
Pesky message_admins(), not worthy of logging either, please remove as it's only useful for testing purposes.